lib/typeErrors.ml

```diff
@@ -624,6 +628,9 @@ let pp_message = function
       ^^^ !^(string_of_int decl)
     in
     { short; descr = Some descr; state = None }
+  | Unexpected_ghost_args_in_function ->
+    let short = !^"Translating functions with ghost arguments not supported" in
+    { short; descr = None; state = None }
```

Comment:
It's more that it's not implemented rather than not supported.

```ocaml
  | Not_impl_ghost_args_in_pure_C_function ->
    let short = !^"Cannot lift a pure C function which uses ghost arguments (not implemented)." in
    { short; descr = None; state = None }
```